src/main/java/com/spotinst/sdkjava/model/bl/aws/managed/instance/Strategy.java

```diff
@@ -0,0 +1,210 @@
+        isSet.add("fallbackToOd");
+        this.fallbackToOd = fallbackToOd;
+    }
+
+    public String getLifeCycle() {
+        return lifeCycle;
+    }
+
+    public void setLifeCycle(String lifeCycle) {
+        isSet.add("lifeCycle");
+        this.lifeCycle = lifeCycle;
+    }
+
+    public List<String> getOptimizationWindows() {
+        return optimizationWindows;
+    }
+
+    public void setOptimizationWindows(List<String> optimizationWindows) {
+        isSet.add("optimizationWindows");
+        this.optimizationWindows = optimizationWindows;
+    }
+
+    public String getOrientation() {
+        return orientation;
+    }
+
+    public void setOrientation(String orientation) {
+        isSet.add("orientation");
+        this.orientation = orientation;
+    }
+
+    public RevertToSpot getRevertToSpot() {
+        return revertToSpot;
+    }
+
+    public void setRevertToSpot(RevertToSpot revertToSpot) {
+        isSet.add("revertToSpot");
+        this.revertToSpot = revertToSpot;
+    }
+
+    public Boolean getUtilizeReservedInstances() {
+        return utilizeReservedInstances;
+    }
+
+    public void setUtilizeReservedInstances(Boolean utilizeReservedInstances) {
+        isSet.add("utilizeReservedInstances");
+        this.utilizeReservedInstances = utilizeReservedInstances;
+    }
+    //endregion
+
+    //region Builder class
+    public static class Builder {
+        private Strategy strategy;
+
+        private Builder() {
+            this.strategy = new Strategy();
+        }
+
+        public static Builder get() {
+            Builder builder = new Builder();
+            return builder;
+        }
+
+        public Builder setDrainingTimeout(final Integer drainingTimeout) {
+            strategy.setDrainingTimeout(drainingTimeout);
+            return this;
+        }
+
+        public Builder setFallbackToOd(final Boolean fallbackToOd) {
+            strategy.setFallbackToOd(fallbackToOd);
+            return this;
+        }
+
+        public Builder setLifeCycle(final String lifeCycle) {
+            strategy.setLifeCycle(lifeCycle);
+            return this;
+        }
+
+        public Builder setOptimizationWindows(final List<String> optimizationWindows) {
+            strategy.setOptimizationWindows(optimizationWindows);
+            return this;
+        }
+
+        public Builder setOrientation(final String orientation) {
+            strategy.setOrientation(orientation);
+            return this;
+        }
+
+        public Builder setRevertToSpot(final RevertToSpot revertToSpot) {
+            strategy.setRevertToSpot(revertToSpot);
+            return this;
+        }
+
+        public Builder setUtilizeReservedInstances(final Boolean utilizeReservedInstances) {
+            strategy.setUtilizeReservedInstances(utilizeReservedInstances);
+            return this;
+        }
+
+        public Strategy build() {
+            // TODO : Validations
```

Comment:
Remove comment. This comment is applicable to all files throughout this document.